xml2rfc/writers/text.py

```diff
@@ -2746,7 +2749,8 @@ def render_reference(self, e, width, **kwargs):
         text += '.'
         for ctag in ('annotation', ):
             for c in e.iterdescendants(ctag):
-                text = self.tjoin(text, c, width, **kwargs)
+                # use MAX_WIDTH here since text gets formatted later
+                text = self.tjoin(text, c, MAX_WIDTH, keep_url=True, **kwargs)
```

Comment:
If we don't already have one, this deserves a test demonstrating what happens if we have annotation content that is larger than width but smaller than MAX_WIDTH going through this.

Reply:
I've added unit tests with 881d8a17ddc226ca9f787eb152cd706b315d338a